flask_restplus_patched/parameters.py

```diff
@@ -1,9 +1,14 @@
 # encoding: utf-8
 # pylint: disable=missing-docstring
-from six import itervalues
+import logging
 
+from six import itervalues
 from flask_marshmallow import Schema, base_fields
-from marshmallow import validate
+from marshmallow import validate, validates_schema
+from marshmallow.exceptions import ValidationError
+
+
+log = logging.getLogger(__name__) # pylint: disable=invalid-name
```

Comment:
Please, use two spaces between code and comments and fix wherever you see is done otherwise.

This way the code will be still readable even with weird or missing syntax highlighting.